DEVELOPERS_GUIDE.md

```diff
@@ -83,12 +83,61 @@ Ensure you have an up to date version of cmake or use cmake3
 
 ## CPP Codeing Style Guide
 
- VOTCA now uses the clang formatter to automatically ensure consistent style.
+ VOTCA uses the clang formatter to automatically ensure consistent style.
  The style is essentially the same, as what is used by google, with the 
- exception that types and = are aligned to make the code more readable.
- Have a look at `.clang-format file` in the 
- [main votca repository](https://github.com/votca/votca/blob/master/.clang-format) for details.
+ exception that types and = are aligned to make the code more readable. 
 
+ The contents of the .clang-format file are shown below:
```

Comment:
Sorry, but why are you adding this back? Please put a link to the file in the repo instead.